src/components/content/TextAreaComponent.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Box, Textarea } from "@chakra-ui/react";
+
+const TextAreaComponent: React.FC = (props) => {
+  return (
+    <Box my={5}>
+      <Textarea
+        borderWidth={0}
+        placeholder="Dodaj plik aby rozpocząć rozpoznawanie tekstu"
+        resize={"none"}
+        size={"lg"}
+        variant={"outline"}
```

Comment:
When passing prop as a string we don't need curly brackets:
e.g. `variant="outline"` or `size="lg"`